### ISSUE TITLE
Find room to work

### ISSUE THREAD

**Issue by JonasPetersenFAU** [amosproj/amos2022ss03-turtlebot-fleet-management#2]
## User story
1. As a team
2. we want / need a room 
3. So that we can begin to work on our project and try out the turtlebots and get used to them

## Acceptance criteria
* Room is big enough to drive turtlebot
* Room can be booked regularly on fixed dates
* Best on ground or first floor
* Place to store the turtlebot

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
